CONTRIBUTING.md

```diff
@@ -32,6 +32,13 @@ maximize the chances of your PR being merged.
   Please see [envoy/support/README.md](https://github.com/envoyproxy/envoy/tree/master/support) for
   more information on these hooks.
 
+  Note: the git hooks depend upon many of the dependencies used to build envoy. Install these on macOS with:
+
+  ```bash
+  brew install coreutils wget cmake libtool go bazel automake ninja clang-format autoconf aspell
```

Comment:
This dependency list can get out of sync with upstream Envoy relatively easily - what do you think about linking to the [upstream's readme section](https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#quick-start-bazel-build-for-developers) instead? This also has the benefit of being non-specific to macOS